### PR TITLE
Purges most clown planet references

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
Removes most references to clown planet within the code and rewrites the accompanying text to accomodate this.

Ruins are unaffected currently but will be purged properly in the future.

## Why It's Good For The Game
By order of the Lore Team, this content has been marked for immediate destruction. 

Clown planet is an ancient meme that has no reason for existing beyond lolrandom funni clown world. The world will be better off for its passing.

## Testing
- Loaded in
- Spawned items
- Descriptions are updated

## Changelog
:cl:
del: Removed clown planet references
tweak: Rewrote several item and mob descriptions
/:cl:

### CODE REVIEW

File: code/game/objects/items/flag.dm [hunk 1]
```diff
@@ -54,13 +54,13 @@
 	icon_state = "ntflag"
 
 /obj/item/flag/clown
-	name = "\improper Clown Planet flag"
-	desc = "The banner of His Majesty, King Squiggles the Eighth."
+	name = "\improper Clown Unity flag"
+	desc = "The universal banner of clowns everywhere. It smells faintly of banana."
```
Comment (DGamerL) [inline code/game/objects/items/flag.dm:58]:
```suggestion
	desc = "The universal banner of clowns everywhere. It smells faintly of bananas."
```

File: code/modules/food_and_drinks/food/foods/baked_goods.dm [hunk 2]
```diff
@@ -277,7 +277,7 @@
 
 /obj/item/reagent_containers/food/snacks/pie
 	name = "banana cream pie"
-	desc = "Just like back home, on clown planet! HONK!"
+	desc = "One of the five essential food groups of clowns everywhere."
```
Comment (DGamerL) [inline code/modules/food_and_drinks/food/foods/baked_goods.dm:280]:
```suggestion
	desc = "One of the five essential food groups of clowns."
```
This fits better to me

File: code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /mob/living/simple_animal/hostile/retaliate/clown
 	name = "Clown"
-	desc = "A denizen of clown planet"
+	desc = "A strange creature that vaguely resembles a normal station clown. Upon closer inspection, it is nothing of the sort."
```
Comment (DGamerL) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm:3]:
```suggestion
	desc = "A strange creature that vaguely resembles a normal clown. Upon closer inspection, it is nothing of the sort."
```
Station =/= universe

**Comment by lewcc** [comment]
solid-snake-salute

**Comment by Danchi299** [comment]
you know what, being born a clown on Clown Planet is one thing
but *deciding* to be a clown and going to specialized Clown University is way funnier

File: code/modules/mob/language.dm [hunk 4]
```diff
@@ -463,7 +463,7 @@
 
 /datum/language/clown
 	name = "Clownish"
-	desc = "The language of clown planet. Mother tongue of clowns throughout the Galaxy."
+	desc = "The language of Clown University. Mother tongue of clowns throughout the Galaxy."
```
Comment (Burzah) [inline code/modules/mob/language.dm:466]:
```suggestion
	desc = "The language of Clown University. Mother tongue of clowns throughout the galaxy."
```
Small nitpick, while we are here. Galaxy in this case doesn't appear to be a proper noun, so it shouldn't be capitalized.

**Comment by S34NW** [comment]
Please merge `upstream/master` into your branch to fix CI failing due to external factors.

**Comment by PopeDaveThe3th** [comment]
> Please merge `upstream/master` into your branch to fix CI failing due to external factors.

Done.

**Review by DGamerL — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**